pmd-apex-jorje/repo/create-local-repo.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-URL=https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/54b127925eebfcfd14cd9988018438ba49ec3d43/packages/salesforcedx-vscode-apex/out/apex-jorje-lsp.jar
-VERSION=2017-11-17
+URL=https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/5e6c90f09c8a5187fffae062b40ecb09ebb7f6a6/packages/salesforcedx-vscode-apex/out/apex-jorje-lsp.jar
+VERSION=2018-11-21
```

Comment:
I was working towards having this script "update" to the latest version from the develop branch... I had:

```bash
LAST_COMMIT_DATE=`curl -s "https://api.github.com/repos/forcedotcom/salesforcedx-vscode/commits?sha=develop&path=packages%2Fsalesforcedx-vscode-apex%2Fout%2Fapex-jorje-lsp.jar&page=1&per_page=1"  | jq -r '.[0].commit.committer.date'`
VERSION=${LAST_COMMIT_DATE%T*}

URL=https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-apex/out/apex-jorje-lsp.jar
FILENAME=apex-jorje-lsp-${VERSION}.jar
FILENAME_MINIMIZED=apex-jorje-lsp-minimized-${VERSION}.jar
```

It works fine (as we are using an unauthorized API request to github, beware of rate limits!), but of course is limited to a single update per day as we use the date as version.

Also, it's still missing changing the `VERSION` in the pom for this module

Reply:
Also, the `install` step uses `-DlocalRepositoryPath=.` which assumes we are running the script from the repo folder. We could update it to ```-DlocalRepositoryPath=`dirname $0` ```

Reply:
Good ideas! I've integrated them :smile: 

What's still open:
* updating the version in the pom. Maybe we can just call `sed` on `$(dirname $0)/../pom.xml`?
* I'd like to remove the old jar's from the repo - there is no use to keep the old binaries around. So, a `git rm` in some way...

Reply:
Sure, sed should be enough.

+1 to removing old versions